LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 NHSX
```

Comment:
On [this repository](https://github.com/nhsx/self-isolation-prototype/blob/master/LICENSE), they have used NHSX.UK as the license copyright. We should probably copy that.

Reply:
Fair, though on this repository it used `NHSX` which I copied (I haven't checked all examples)

https://github.com/nhsx/nhsx-website/blob/master/LICENSE#L3

Reply:
ah, then that's fine.

Reply:
Yep this should be NHSX 👍